### ISSUE TITLE
【EESS】New submissions for Thu, 25 May 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#229]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### Open Source High Fidelity Modeling of a Type 5 Wind Turbine Drivetrain  for Grid Integration
 - **Authors:** Tanveer Hussain, Juan Gallego-Calderon, S M Shafiul Alam
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2305.15216
 - **Pdf link:** https://arxiv.org/pdf/2305.15216
 - **Abstract**
 The increasing integration of renewable energy resources in evolving bulk power system (BPS) is impacting the system inertia. Type-5 wind turbine generation has the potential to behave like a traditional synchronous generator and can help improve system inertia. Hydraulic torque converter (TC) and gearbox with torque limiting feature are integral parts of a Type-5 wind turbine unit. High fidelity model of Type-5 wind turbine including these core components is not openly and widely available for grid integration and transient stability studies. This hinders appropriate assessment of Type-5 wind power plant's contribution to bulk grid resilience. This work develops a TC model based on those generally used in automobile's transmission system. Moreover, the concept of torsional coupling is leveraged to integrate the TC and gearbox system dynamics. The entire integrated model will be open sourced and publicly available for grid integration studies.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Sample-Efficient Learning for a Surrogate Model of Three-Phase  Distribution System
 - **Authors:** Hoang Tien Nguyen, Young-Jin Kim, Dae-Hyun Choi
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2305.14799
 - **Pdf link:** https://arxiv.org/pdf/2305.14799
 - **Abstract**
 A surrogate model that accurately predicts distribution system voltages is crucial for reliable smart grid planning and operation. This letter proposes a fixed-point data-driven surrogate modeling method that employs a limited dataset to learn the power-voltage relationship of an unbalanced three-phase distribution system. The proposed surrogate model is designed using a fixed-point load-flow equation, and the stochastic gradient descent method with an automatic differentiation technique is employed to update the parameters of the surrogate model using complex power and voltage samples. Numerical examples in IEEE 13-bus, 37-bus, and 123-bus systems demonstrate that the proposed surrogate model can outperform surrogate models based on the deep neural network and Gaussian process regarding prediction accuracy and sample efficiency
### Truncated Turbo Equalizer with SIC for OTFS
 - **Authors:** Sanoopkumar P. S., Stephen McWade, Arman Farhang
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2305.14966
 - **Pdf link:** https://arxiv.org/pdf/2305.14966
 - **Abstract**
 Orthogonal time frequency space (OTFS) is a promising candidate waveform for the next generation wireless communication systems. OTFS places data in the delay-Doppler (DD) domain, which simplifies channel estimation in highmobility scenarios. However, due to the 2-D convolution effect of the time-varying channel in the DD domain, equalization is still a challenge for OTFS. Existing equalizers for OTFS are either highly complex or they do not consider intercarrier interference present in high-mobility scenarios. Hence, in this paper, we propose a novel two-stage detection technique for coded OTFS systems. Our proposed detector brings orders of magnitude computational complexity reduction compared to existing methods. At the first stage, it truncates the channel by considering only the significant coefficients along the Doppler dimension and performs turbo equalization. To reduce the computational load of the turbo equalizer, our proposed method deploys the modified LSQR (mLSQR) algorithm. At the second stage, with only two successive interference cancellation (SIC) iterations, our proposed detector removes the residual interference caused by channel truncation. To evaluate the performance of our proposed truncated turbo equalizer with SIC (TTE-SIC), we set the minimum mean squared error (MMSE) equalizer without channel truncation as a benchmark. Our simulation results show that the proposed TTE-SIC technique achieves about the same bit error rate (BER) performance as the benchmark.
### Bistatic OFDM-based Joint Radar-Communication: Synchronization, Data  Communication and Sensing
 - **Authors:** Lucas Giroto de Oliveira, David Brunner, Axel Diewald, Charlotte Muth, Laurent Schmalen, Thomas Zwick, Benjamin Nuss
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2305.15058
 - **Pdf link:** https://arxiv.org/pdf/2305.15058
 - **Abstract**
 This article introduces a bistatic joint radar-communication (RadCom) system based on orthogonal frequency-division multiplexing (OFDM). In this context, the adopted OFDM frame structure is described and system model encompassing time, frequency, and sampling synchronization mismatches between the transmitter and receiver of the bistatic system is outlined. Next, the signal processing approaches for synchronization and communication are discussed, and radar sensing processing approaches using either only pilots or a reconstructed OFDM frame based on the estimated receive communication data are presented. Finally, proof-of-concept measurement results are presented to validate the investigated system and a trade-off between frame size and the performance of the aforementioned processing steps is observed.
### Power Reduction Opportunities on End-User Devices in Quality-Steady  Video Streaming
 - **Authors:** Christian Herglotz, Werner Robitza, Alexander Raake, Tobias Hossfeld, André Kaup
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2305.15117
 - **Pdf link:** https://arxiv.org/pdf/2305.15117
 - **Abstract**
 This paper uses a crowdsourced dataset of online video streaming sessions to investigate opportunities to reduce the power consumption while considering QoE. For this, we base our work on prior studies which model both the end-user's QoE and the end-user device's power consumption with the help of high-level video features such as the bitrate, the frame rate, and the resolution. On top of existing research, which focused on reducing the power consumption at the same QoE optimizing video parameters, we investigate potential power savings by other means such as using a different playback device, a different codec, or a predefined maximum quality level. We find that based on the power consumption of the streaming sessions from the crowdsourcing dataset, devices could save more than 55% of power if all participants adhere to low-power settings.
### Solving Diffusion ODEs with Optimal Boundary Conditions for Better Image  Super-Resolution
 - **Authors:** Yiyang Ma, Huan Yang, Wenhan Yang, Jianlong Fu, Jiaying Liu
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2305.15357
 - **Pdf link:** https://arxiv.org/pdf/2305.15357
 - **Abstract**
 Diffusion models, as a kind of powerful generative model, have given impressive results on image super-resolution (SR) tasks. However, due to the randomness introduced in the reverse process of diffusion models, the performances of diffusion-based SR models are fluctuating at every time of sampling, especially for samplers with few resampled steps. This inherent randomness of diffusion models results in ineffectiveness and instability, making it challenging for users to guarantee the quality of SR results. However, our work takes this randomness as an opportunity: fully analyzing and leveraging it leads to the construction of an effective plug-and-play sampling method that owns the potential to benefit a series of diffusion-based SR methods. More in detail, we propose to steadily sample high-quality SR images from pretrained diffusion-based SR models by solving diffusion ordinary differential equations (diffusion ODEs) with optimal boundary conditions (BCs) and analyze the characteristics between the choices of BCs and their corresponding SR results. Our analysis shows the route to obtain an approximately optimal BC via an efficient exploration in the whole space. The quality of SR results sampled by the proposed method with fewer steps outperforms the quality of results sampled by current methods with randomness from the same pretrained diffusion-based SR model, which means that our sampling method ``boosts'' current diffusion-based SR models without any additional training.
